apps/sensenet/src/components/drawer/PermanentDrawer.tsx

```diff
@@ -67,11 +67,26 @@ const useStyles = makeStyles((theme: Theme) => {
   })
 })
 
+const isLocalization = (pathname: string) => {
+  const res = decodeURIComponent(pathname).split('/')
```

Comment:
is this 2 function needed? can't we use the matchPath here as well?

Reply:
Yes, we need them, cause in these cases we encode the Url datas (use-drawer-items.tsx line 83-133) We cannot get these info any other way. 
After the URL refactor, we should change this logic here as well (so it is just a temporary solution)

Reply:
The perfect solution was to create new componets for localization and users and groups with their own URL